index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
+    <title>Marcelo Alarcon Portafolio</title>
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="images/codeoutlinedprogrammingsigns_81143.ico"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <img src="images/Welcome.png" alt="logo" />
+      <img src="images/Union.png" alt="menu button" />
+    </header>
+
+    <section class="headline">
+      <div class="presentation_container">
+        <h2 class="primary_color">Hey there. I'm Marcelo</h2>
```

Comment:
- [ ] I think it would be better to use ```h1``` The main title instead of using ```h2```  (e.g., "Hey there. I'm Marcelo") is a  ```h1``` tag. :joy:

    The ```H1``` tag  is an HTML heading that's most commonly used to mark up a web page title :+1: :+1: 

    _If you want that google (and other search engines) know about a specific topic, in this case, you, use an ```h1``` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info._